### ISSUE TITLE
sqlite3.dll not included

### ISSUE THREAD

**Issue by Athanasius** [albertosottile/py2exe#23]
Using  py2exe-0.9.3.2-cp37-none-win32.whl I'm not seeing sqlite3.dll included when having:

```
{ 'py2exe':
  {
    'packages': [
      'sqlite3'
    ]
  }
}
```
in the `options` passed to setup().  Files like `_sqlite3.pyc` and `sqlite3/dump.pyc` are making it into the library.zip, but not the .dll from `sys.base_prefix`\DLLs\sqlite3.dll.

I guess something needs adding to hooks.py to pick it up?  Literally the only mention of 'dll' in the full setup.py is to include an unrelated local copy of WinSparkle.dll.  No other mentions of `sqlite` in setup.py either.

**Comment by albertosottile** [comment]
Thanks for reporting this. After some testing, I can confirm that this is a bug in the current codebase (interestingly, this bug does not occur inside `conda` environments, hence the initial mislabeling). 

Investigations pointed out that the system embedded in `py2exe` that checks for external dependencies of loaded libraries (mostly in [py2exe/dllfinder.py](py2exe/dllfinder.py)) does not work anymore with several modules in the standard library (including `sqlite3`).

Therefore, this bug could be fixed either by adding a custom hook for `sqlite3` that manually retieves and packs the missing DLL, or by patching this broken system. At the moment, I am inclined to do the latter, but such a fix will probably be released in 0.9.4.0 since it will heavily change a key component of `py2exe`.

**Comment by albertosottile** [comment]
This issue is now fixed in master and it will be fixed in the next release (0.9.4.0). In the meantime, if you could test it on your system with latest wheels from [Bintray](https://bintray.com/alby128/py2exe/py2exe/0.9.4.0#files), that would be great.

**Comment by Athanasius** [comment]
Sorry it took a week to get to this.  Yes, 0.9.4.0 correctly includes sqlite3.dll without me forcing it in setup.py.  The resulting binary then works after .msi packaging and installation.
